src/test/kotlin/com/zepben/evolve/cim/iec61970/base/core/EquipmentContainerTest.kt

```diff
@@ -40,4 +40,34 @@ internal class EquipmentContainerTest {
             EquipmentContainer::clearEquipment
         )
     }
+
+    @Test
+    internal fun normalFeeders() {
+        val fdr1 = Feeder()
+        val fdr2 = Feeder()
+        val fdr3 = Feeder()
+
+        val eq1 = object : Equipment() {}.addContainer(fdr1).addContainer(fdr2)
+        val eq2 = object : Equipment() {}.addContainer(fdr2).addContainer(fdr3)
+
+        val equipmentContainer = object : EquipmentContainer() {}.addEquipment(eq1).addEquipment(eq2)
+
+        assertThat(equipmentContainer.normalFeeders(), containsInAnyOrder(fdr1, fdr2, fdr3))
+        assertThat(equipmentContainer.currentFeeders(), empty())
+    }
+
+    @Test
+    internal fun currentFeeders() {
+        val fdr1 = Feeder()
+        val fdr2 = Feeder()
+        val fdr3 = Feeder()
+
+        val eq1 = object : Equipment() {}.addCurrentContainer(fdr1).addCurrentContainer(fdr2)
+        val eq2 = object : Equipment() {}.addCurrentContainer(fdr2).addCurrentContainer(fdr3)
+
+        val equipmentContainer = object : EquipmentContainer() {}.addEquipment(eq1).addEquipment(eq2)
+
+        assertThat(equipmentContainer.currentFeeders(), containsInAnyOrder(fdr1, fdr2, fdr3))
+        assertThat(equipmentContainer.normalFeeders(), empty())
```

Comment:
Some of the comments in my previous review will be resolved by these tests. Can you also reverse the order of these to keep them consistent with the previous test. I had to have a second read to realise you had swapped the order as I thought it was confirming the same thing as the previous test originally.